install_script.sh.template

```diff
@@ -1253,9 +1254,7 @@ fi
 
 # set the system-probe configuration
 if [ -n "$system_probe_ensure_config" ]; then
-  if ! ensure_config_file_exists "$sudo_cmd" "$system_probe_config_file" "root"; then
-    echo "System probe config file already created"
-  fi
+  ensure_config_file_exists "$sudo_cmd" "$system_probe_config_file" "root"
```

Comment:
I think that with this, the system probe config file will never be created since the `valid_condition` variable is not passed, right?

Reply:
Maybe something like that could work
```
ensure_config_file_exists "$sudo_cmd" "$system_probe_config_file" "root" $system_probe_ensure_config
```
After deleting the if condition

Reply:
yes I need to change this now I changed the condition in the method, thanks a lot

Reply:
revert the changes on `ensure_config_file_exists` and changed the `update_runtime_security` and `update_compliance_configuration` methods
We should be back in the initial situation now, thanks for your help and sorry for inconvenience